app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'user_id'
+  has_many :likes, foreign_key: 'user_id'
+
```

Comment:
- [Optional] Good job using the `author_id` for the association between the `post` and `user` models, but the same structure should be preserved for the `comments` and `likes` association. Please consider following the same structure and changing the `user_id` to `authot_id`.